Documentation/hypernode-api/settings/README.md

```diff
@@ -27,3 +27,5 @@ this setting see [this changelog](https://support.hypernode.com/changelog/releas
 - `blackfire_enabled`: boolean - Indicates whether BlackFire is enabled on your node.
 - `blackfire_server_id`: - string - The BlackFire server id for your blackfire setup.
 - `blackfire_server_token`: string - The BlackFire server token for your blackfire setup.
+- `openvpn_enabled`: boolean - Enable or disable the [OpenVPN on Hypernode](/Documentation/hypernode-vpn/README.md)
```

Comment:
You can leave the "the" out. As well as `on Hypernode` since these are the hypernode docs :p